### PR TITLE
Add link to Spanish version of Getting Started document.

### PR DESCRIPTION
**Pull request by emcelroy**

### CODE REVIEW

File: src/microsite/getting-started/index.html [hunk 1]
```diff
@@ -98,7 +98,7 @@ <h2>Dynamic time-based modeling</h2>
...
 <h2>Tutorial</h2>
-<p>Try a step-by-step tutorial by opening this <a href="https://sagemodeler.concord.org/#file=examples:Getting%20Started" target="_blank" rel="noopener">Getting Started document</a> and following the instructions.</p>	</div>
+<p>Try a step-by-step tutorial by opening this <a href="https://sagemodeler.concord.org/#file=examples:Getting%20Started" target="_blank" rel="noopener">Getting Started document</a> and following the instructions. Also available <a href="https://sagemodeler.concord.org/app/?lang=es#file=examples:Gu%C3%ADa%20Comenzando%20-%20Getting%20started%20in%20Spanish" target="_blank" rel="noopener">en espa&ntilde;ol</a>.</p>	</div>
```
Comment (dougmartin) [inline src/microsite/getting-started/index.html:101]: I'm getting an "Unable to load" error when I try the link you've added (https://sagemodeler.concord.org/app/?lang=es#file=examples:Gu%C3%ADa%20Comenzando%20-%20Getting%20started%20in%20Spanish)

**Review by dougmartin — CHANGES_REQUESTED** [review]
Can you check the link you added to ensure it loads?

File: src/microsite/getting-started/index.html (hunk 1 above)
Comment (emcelroy) [reply src/microsite/getting-started/index.html:101]: @dougmartin Weird. I just checked it and it's working for me. Is it still not working for you? 

image

**Comment by ddamelin** [comment]
So I experienced the same load error thing until I closed all open SageModeler tabs and cleared cache. (I noticed it would work fine in an incognito window before that.) Not sure what it going on that would cause that though. Cached old CFM code?

**Comment by dougmartin** [comment]
@emcelroy / @ddamelin - yep it loaded find when I shift+reloaded it to clear the cache.

File: src/microsite/getting-started/index.html (hunk 1 above)
Comment (dougmartin) [reply src/microsite/getting-started/index.html:101]: Fixed with cache clear.  Looks good :+1:

**Pull request merged**